src/assets/svg/testimonial-mux-logo.svg

```diff
@@ -0,0 +1,13 @@
+<svg width="124" height="40" viewBox="0 0 124 40" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path opacity="0.5" d="M61.5089 30.7579C66.796 30.7579 71.0819 26.4764 71.0819 21.195V4.34932C71.0819 1.96395 73.0176 0.0304375 75.4056 0.0304375C77.7933 0.0304375 79.7292 1.96395 79.7292 4.34932V21.0633C79.7292 31.1109 71.5964 39.2594 61.5483 39.3018C61.5307 39.302 61.5134 39.3032 61.4957 39.3032C59.1394 39.2961 57.2315 37.3863 57.2315 35.0308C57.2315 32.671 59.1465 30.7579 61.5089 30.7579Z" fill="#2D4D55"/>
```

Comment:
instead of `fill="#2D4D55"`, this should be `class="fillStd"`, so that the styling and hover effects are consistent across icons

see how it's done here:
https://github.com/tilt-dev/tilt.build/blob/master/src/assets/svg/testimonial-yext-logo.svg?short_path=5a075c3